README.md

```diff
@@ -2,12 +2,12 @@
 
 <h1 align="center">
   <br>
-  <a href="https://github.com/Abdur-rahmaanJ"><img src="https://github.com/Abdur-rahmaanJ/shopyo/blob/dev/github_banner.png" alt="shopyo" width="" height=""></a>
+  <a href="https://github.com/shopyo"><img src="https://github.com/shopyo/shopyo/blob/dev/github_banner.png" alt="shopyo" width="" height=""></a>
   
   
 </h1>
 
-[![Codecov](https://codecov.io/gh/Abdur-rahmaanJ/shopyo/branch/dev/graph/badge.svg?token=J4TL2MDTSS)](https://codecov.io/gh/Abdur-rahmaanJ/shopyo) ![Tests](https://github.com/Abdur-rahmaanJ/shopyo/actions/workflows/tests.yaml/badge.svg) [![PyPI version shields.io](https://img.shields.io/pypi/v/shopyo.svg)](https://pypi.python.org/pypi/shopyo/) [![Documentation Status](https://readthedocs.org/projects/shopyo/badge/?version=latest)](https://shopyo.readthedocs.io/en/latest/?badge=latest)
+[![Codecov](https://codecov.io/gh/Abdur-rahmaanJ/shopyo/branch/dev/graph/badge.svg?token=J4TL2MDTSS)](https://codecov.io/gh/Abdur-rahmaanJ/shopyo) ![Tests](https://github.com/shopyo/shopyo/actions/workflows/tests.yaml/badge.svg) [![PyPI version shields.io](https://img.shields.io/pypi/v/shopyo.svg)](https://pypi.python.org/pypi/shopyo/) [![Documentation Status](https://readthedocs.org/projects/shopyo/badge/?version=latest)](https://shopyo.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Thank you. You forgot to change the codecov bade. It should be this `[![codecov](https://codecov.io/gh/shopyo/shopyo/branch/dev/graph/badge.svg?token=HKA0BQ76XD)](https://codecov.io/gh/shopyo/shopyo)`

Reply:
i fixed the badge